holochain-introduction-for-developers/holochain-brief-introduction.md

```diff
@@ -0,0 +1,75 @@
+# What is Holochain?
+
+> Holochain is a _protocol_ and _framework_ for creating cryptographically secure, portable, and distributed applications. It provides the necessary tools to allow individuals to maintain control over their identity and data, while interacting safely with a network of untrusted peers.
+>
+> The purpose of this document is to help developers and technologists quickly understand:
+>
+> * Why we're building Holochain
+> * What the project consists of
+> * What advantages it has over server-based and blockchain-based architectures
+
+## The problem
+
+The Internet is decentralized by design. Many services, such as the web and email, have no central points. But in order to move through the Internet confidently, we need **trust.** How can you know I am who I say I am, and how can you be sure that I'm acting with integrity?
+
+Centralized services act as proxies of trust. We may not be able to trust a million anonymous peers, but we can trust these big services to faithfully negotiate our interactions with those peers — **until we can't.** These systems are beginning to show weaknesses that are direct consequences of centralization.
+
+* **They are more attractive to attackers.**
+* **They leave users vulnerable to service outages.**
+* **They concentrate power in the hands of a few.**
+* **They burden the service operator with liabilities and costs.**
+
+Blockchain opened our imaginations to the possibility of trustable, decentralized coordination with untrusted peers, but suffers from efficiency problems that are inherent to its design. The next wave of decentralized ledgers improve performance and resilience but carry over some of blockchain's assumptions. They hint at the full potential of this technology but stop before they reach the finish line.
+
+## Holochain: fully sovereign, distributed, trustable coordination
+
+Holochain gives you the tools to create applications that enable direct, trustable interactions among sovereign peers. It does this without needing the central coordination of a server or a group of miners. We are building:
+
+* A **peer-to-peer protocol** that shares data among participants, giving them the power to act collectively as an 'immune system' that detects invalid data and pushes out bad actors.
+* An open-source **application development framework** that abstracts away complexity and allows you to focus on your core functionality.
+* A set of **core apps and utilities**, including an identity manager, a private data vault, and a software distribution platform.
+* A **distributed hosting ecosystem** called Holo Hosting which bridges Holochain to the web so people can use your apps as if they were traditional, server-based apps.
+
+### Uncompromisable sovereignty
```

Comment:
I hope for the inclusion of the term "mutual" where ever we use sovereign, because I wish for us to appeal not only to individual agency, but deep cooperation in the context of that agency.  Sovereignty by itself is just the power to say #@!$#*%&# off!  But when that is landed in the context of mutuality something more powerful and transcendent happens.   Remember Holochain is BOTH agent centric and socio-centric, in that we are all validating the rules together, and that social act is what creates the power of sovereignty.  I'm not saying you have to change a word.  I'm just trying to point to what to me is a crucial part of the system.

Reply:
Yes, this represents a bit of a struggle for me. On the one hand, I know it's our job to shift the Overton window, but at the same time I don't want to use too much Holochain-ese, know what I mean? I don't want to lose the fact that this is about _interdependence_, not _independence_, but I'm already struggling to be brief :D